play.html

```diff
@@ -115,6 +111,12 @@
 
 
                 case 'backspace':
+                    document.getElementById('L1').className = 'blank'
+                    document.getElementById('L2').className = 'blank'
+                    document.getElementById('L3').className = 'blank'
+                    document.getElementById('L4').className = 'blank'
+                    document.getElementById('L5').className = 'blank'
```

Comment:
These 5 lines and the 5 lines :91-95 are almost identical. Try moving them to a separate function with a nice name:
```javascript
setClassNameForLetters(className) {
    for (let i = 1; i <= 5; i++) {
        document.getElementById(`L${i}`).className = className
    }
}
```

Reply:
done